src/components/App.js

```diff
@@ -14,6 +14,13 @@ var SUB = new NchanSubscriber(
 export default class App extends React.Component {
   constructor(props) {
     super(props);
+    /** *
+     * Loads user volume preference
+     */
+    let volume;
+    if (!localStorage.getItem("coderadio-volume")) volume = 0.5;
+    else volume = localStorage.getItem("coderadio-volume") / 10;
```

Comment:
you can keep volume as a state and do something similar to the following:

volume : localStorage.getItem("coderadio-volume") ? localStorage.getItem("coderadio-volume") / 10 : 0.5